lib/net/dhcp/record.rb

```diff
@@ -10,8 +10,16 @@ def initialize(opts = { })
       self.ip      = Net::Validations.validate_ip! self.ip
     end
 
+    def lease?
+      type && type == "lease"
```

Comment:
Yeah, my thinking was if you setup Foreman with older version of Proxy, it will not return type in the REST API response. Type will be `nil` but it is not necessary here, sure.